### PR TITLE
CAUTH-32 Add support for GET

### PR DESCRIPTION
**Pull request by kusold**
This adds support for the `GET` method that was added in limitd/limitdb#11

### CODE REVIEW

File: test/protocol.tests.js [hunk 1]
```diff
@@ -67,6 +67,27 @@ describe('protocol', function () {
       assert.deepEqual(decoded.take.limit, 30);
```
[the remark this answers is not in the document]
Comment (kusold) [reply test/protocol.tests.js]: Followed the pattern for the current backwards compatibility tests. d8d770f

[the remark this answers is not in the document]
Comment (kusold) [reply test/protocol.tests.js]: Reworked the compatibility test. It now does:

Encode with 2.0.5 -> Decode with current
Encode with current -> Decode with 2.0.5

File: test/backward/v2.0.5/Index.proto [hunk 2]
```diff
@@ -0,0 +1,6 @@
+syntax = "proto2";
```
Comment (jfromaniello) [inline test/backward/v2.0.5/Index.proto]: not a biggy but I think you copied this folder, right? what I did on the past was to create a gitsubmodule pointing to the

Comment (kusold) [reply test/backward/v2.0.5/Index.proto]: That's 100x better than my hacky solution. I'll follow that pattern.

**Pull request merged**